src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -421,7 +428,7 @@ protected function build(ContainerBuilder $container)
      */
     protected function getContainerClass()
     {
-        return $this->name.ucfirst($this->environment).($this->debug ? 'Debug' : '').'ProjectContainer';
+        return 'app'.ucfirst($this->environment).($this->debug ? 'Debug' : '').'ProjectContainer';
```

Comment:
shouldn't we keep using it for now, for BC ? Otherwise, it would be a BC break for people using multiple kernels (with different names) and expecting to have the container classes not clashing together.

Reply:
You mean when a multi-kernel app share the same cache?

Reply:
I don't think this is worth it. As of Symfony 4.x, we are only supporting Flex (so, doing it in 3.4 was not possible, but I think this is totally legitimate now).

Reply:
ok, is it better now?